code/_globalvars/lists/mobs.dm

```diff
@@ -6,6 +6,11 @@ GLOBAL_LIST_EMPTY(deadmins)							//all ckeys who have used the de-admin verb.
 GLOBAL_LIST_EMPTY(directory)							//all ckeys with associated client
 GLOBAL_LIST_EMPTY(stealthminID)						//reference list with IDs that store ckeys, for stealthmins
 
+GLOBAL_LIST_INIT(dangerous_turfs, typecacheof(list(
+	/turf/open/lava,
+	/turf/open/chasm)))
```

Comment:
openspace, space and maybe fakepits/fakespace too?

Reply:
sure

Reply:
ok this being a global list is dumb it should be dynamically generated because some mobs ignore chasms or lava or stuff